dotcom-rendering/src/web/lib/useApi.tsx

```diff
@@ -40,6 +40,6 @@ export const useApi = <T,>(
 	return {
 		data,
 		error,
-		loading: !error && !data,
+		loading: typeof url !== 'undefined' && !error && !data,
```

Comment:
Would `url !== undefined` do the same thing?

Reply:
oh yes that's right, thanks!

Reply:
I'd actually quite like to make `url` mandatory, with a type of `string | null`, because it feels like this fits better with the SWR approach, but that feels like a riskier change if we don't have a great overview of how the hook is used, right?

Anyway, I've fixed the current implementation in line with your suggestion, thanks!